### PR TITLE
Fixes and Images, Gif File Updates

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/bot-tag-mention/csharp/README.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ This sample app demonstrates the use of tag mention funtionality in teams scope
 ### 4. Setup Manifest for Teams
 
 1. Modify the `manifest.json` in the `/AppManifest` folder and replace the following details:
-   - `<<YOUR-MICROSOFT-APP-ID>>` with Application id generated from Step 3
+   - `3a0f426b-7465-4b81-ab2f-d3e4b1d999d0` with Application id generated from Step 3
```
Comment (Pawank-MSFT) [inline samples/bot-tag-mention/csharp/README.md:129]: This should be remove in the PR.

**Review by Pawank-MSFT — COMMENTED** [review]
Please fix small comment

File: samples/bot-tag-mention/csharp/README.md (hunk 1 above)
Comment (Mohammed-MSFT) [reply samples/bot-tag-mention/csharp/README.md:129]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**